packages/app/src/modules/layoutValidation.js

```diff
@@ -118,34 +58,36 @@ const validateYearOverYearLayout = layout => {
             typeof layout[BASE_FIELD_YEARLY_SERIES][0] === 'string'
         )
     ) {
-        throw new Error(errorLabels.yearOverYear.seriesPeriod)
+        throw new NoSeriesError()
     }
 
-    validateAxis(layout.rows, errorLabels.yearOverYear.categoryPeriod)
-
-    validateAxis(layout.columns, errorLabels.yearOverYear.dx)
+    validateAxis(layout.rows, new NoCategoryError())
 }
 
 const validatePieLayout = layout => {
-    validateAxis(layout.columns, errorLabels.defaultSeries)
-    validateAxis(layout.filters, errorLabels.pie.filter)
+    validateAxis(layout.columns, new NoSeriesError())
+    //validateAxis(layout.filters, errorLabels.pie.filter) //FIXME: Redundant?
```

Comment:
Redundant? Pie cannot have an empty filter axis for other reasons anyway (Data + Period means at least 2 dimensions, spread across the 2 available axes).

Reply:
Yep, not needed anymore (no categories in pie charts now)